package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "WCAG-EM-Report-Tool",
-    "version": "3.0.0",
```

Comment:
Maybe we should leave it at 3.0.0 until we release, eg when this tool goes live we call it 3.0.0, and then we start counting?